source/server/connection_handler_impl.cc

```diff
@@ -502,7 +532,7 @@ void ConnectionHandlerImpl::ActiveTcpListener::newConnection(
     active_connection->connection_->addConnectionCallbacks(*active_connection);
     LinkedList::moveIntoList(std::move(active_connection), active_connections.connections_);
   }
-}
+} // namespace Server
```

Comment:
revert? I think clang-format probably did this when you didn't want it.

Reply:
clang-format doesnt play well with `#ifdefs` so it adds it. Now with the constexpr function it formats correctly

Reply:
fixed in 583512a9750f17a04e7c1de4d0a428588fda08a1